api/group.go

```diff
@@ -0,0 +1,16 @@
+/*
+Copyright 2022.
+
+Licensed under the Apache License, Version 2.0 (the "License");
```

Comment:
Not a blocker but I'm noticing the license headers are slightly different than what we have in other files (maybe this existed before for CRDs too). Usually there's a reference to the "Ratify Authors"

Reply:
thanks for calling it out! I'll check through all the license headers.